src/filebuilders/xml_builder.py

```diff
@@ -12,6 +12,7 @@ def build(self, file_number, data, upload_to_google_drive=False):
         output_dir = self.get_output_directory()
         file_name = self.get_file_name()
         root_element_name = self.get_root_element_name()
+        item_name_func = lambda x : self.get_item_name()
```

Comment:
@GalaJoe on the offchance you aren't sure how to get around this hound error - you can reassign function names in python directly rather than using lambda syntax, so in this case you change line 15 to:

`item_name_func = self.get_item_name`

and the rest of the code should work. The `dicttoxml` documentation uses lambda in its example because it isn't just reusing an existing function but defining a slightly new one on the fly, so its the simplest way to do it in that case.